### PR TITLE
Add Gradle logic to allow release publication

### PR DESCRIPTION
**Pull request by oliverlockwood**
Note that `gradle.properties` config on the build machine is required to provide the URL, username and password for an actual release.

### CODE REVIEW

**Comment by oliverlockwood** [comment]
We could always manually bump the version to 0.8, manually upload the artifact to Nexus, and raise a PR to bump the development version to 0.9-SNAPSHOT if that's preferable to polluting the build.gradle history.

**Comment by chbatey** [comment]
Would you mind changing it to use the axion release plugin? that is the one we're standardising on across our libs

**Comment by chbatey** [comment]
You'll need to put dummy defaults on so it passes on travis and on a fresh checkout e.g https://github.com/scassandra/scassandra-server/blob/master/server/build.gradle

**Comment by oliverlockwood** [comment]
@chbatey I've updated to use axion, and in fact that's let me release from a branch (which I've done - apologies, I tried running it to check things were generally working, but expected the release operation to fail!)  That does at least unblock us to use the new version 0.8.0 in UMV though :smiley: 

I'm not entirely clear which part of the scassandra-server build.gradle you're referring to which would ensure this passes on Travis etc.  Could you clarify, please?

**Comment by chbatey** [comment]
```
if (!project.hasProperty("ossrhUsername")) {
    ext.ossrhUsername = "dummy"
}
if (!project.hasProperty("ossrhPassword")) {
    ext.ossrhPassword = "dummy"
}
```

it means if ppl don't have the variables defined they can still build but not release

**Comment by chbatey** [comment]
that example is for a user/pass for maven central

**Comment by oliverlockwood** [comment]
@chbatey gotcha, cheers, not sure how I missed that.

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    ./gradlew publishArtifactPublicationToOpenSourceRepository
+
+To bump the version (when adding functionality or breaking backwards compatibility):
+
+    ./gradlew release -Prelease.forceVersion=0.3.0
+    ./gradlew publishArtifactPublicationToOpenSourceRepository
+
+Get the current version:
+
+    ./gradlew currentVersion
```
Comment (chbatey) [inline RELEASING.md]: worth documenting it won't work unless you define the vars?

Comment (oliverlockwood) [inline RELEASING.md]: Agreed, and done.

**Comment by jsravn** [comment]
resolves #6

**Comment by jsravn** [comment]
Can you remove the 0.8.0 release that got created on the github project?

**Comment by jsravn** [comment]
We shouldn't be tagging the repo until we are releasing this to jcenter proper. Make sure when you run axion to disable the pushing of tags.

**Pull request merged**